api/app/controllers/mno_enterprise/jpi/v1/admin/users_controller.rb

```diff
@@ -182,7 +196,15 @@ def update_sub_tenant(user)
     end
 
     def authorize_user_for_support_role
-      authorize! :read, @user if current_user.support?
+      (authorize! :read, @user) if current_user.support?
+    end
+
+    def authorized_support_user_search
+      # Only allow support users to search by name.
+      return false unless params[:terms].present?
+      first_name = JSON.parse(params[:terms])["name.like"]
```

Comment:
Should this be an exact match?
See comment in JIRA

Reply:
@ouranos Responded in JIRA.
I raised same issue with celine, who talked to marlin. She assured me it was a non-issue.